release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -1207,6 +1211,8 @@ In the following tables, features are marked with the following statuses:
 
 [id="ocp-telco-core-4-16-known-issues_{context}"]
 
+* When a worker node's Topology Manager policy is changed, the NUMA-aware secondary pod scheduler does not respect this change, which can result in incorrect scheduling decisions and unexpected topology affinity errors. As a workaround, restart the NUMA-aware scheduler by deleting the NUMA-aware scheduler pod.
```

Comment:
@ffromani - can you review this known issue text please?

Reply:
LGTM